test/integration/http_integration.h

```diff
@@ -30,9 +30,22 @@ class IntegrationCodecClient : public Http::CodecClientProd {
   void sendData(Http::StreamEncoder& encoder, uint64_t size, bool end_stream);
   void sendTrailers(Http::StreamEncoder& encoder, const Http::HeaderMap& trailers);
   void sendReset(Http::StreamEncoder& encoder);
+  void onStreamDestroy() override {
+    num_active_requests_--;
+    if (num_active_requests_ == 0) {
+      enableIdleTimer();
```

Comment:
Do we want idle timeout behavior in the integration codec client? I think probably not?

Reply:
Initially it was required. Now with the change i am making to enable it in the constructor itself it may not be required. will test and remove it.